code/lib/cli/src/sandbox-templates.ts

```diff
@@ -528,6 +532,156 @@ const internalTemplates = {
     isInternal: true,
     skipTasks: ['bench'],
   },
+  'internal/npm/empty/react-vite/default-ts': {
```

Comment:
Do we still need these sandboxes now that we have your new CI task above?

Reply:
We don't need them 😁 I just didn't want to delete all the configs until we were sure we didn't want them